clients/src/main/java/org/apache/kafka/common/metrics/JmxReporter.java

```diff
@@ -160,17 +160,30 @@ private void reregister(KafkaMbean mbean) {
         }
     }
 
-    private static class KafkaMbean implements DynamicMBean {
+    public int mbeansQty() {
```

Comment:
Yes, we don't use `Qty` as a suffix. We sometimes use `num` prefix and other times `Count` suffix (it would be nice to be consistent, but we're not there yet).